src/index.js

```diff
@@ -0,0 +1,131 @@
+//Wave 1
+"use strict";
+
+//Wave 2
+let currentTemp = 0;
+const tempNumber = document.getElementById('temp-number');
+const landscapeImg = document.getElementById('landscape-img');
+
+document.getElementById('increase-temp').addEventListener('click',()=>{
+    currentTemp += 1;
+    tempNumber.textContent = currentTemp;
+    changeTempColor();
+})
+
+document.getElementById('decrease-temp').addEventListener('click',()=>{
+    currentTemp -= 1;
+    tempNumber.textContent = currentTemp;
+    changeTempColor();
+})
+
+//Feels like section is our landscape.
+const changeTempColor = () => {
+    if (currentTemp >=80) {
+        tempNumber.style.color = 'red';
+        landscapeImg.src = 'assets/Very hot 2.jpeg';
+    } else if (currentTemp >= 70) {
+        tempNumber.style.color = 'orange';
+        landscapeImg.src = 'assets/Humid.jpeg';
+    } else if (currentTemp >= 60) {
+        tempNumber.style.color = 'blue';
+        landscapeImg.src = 'assets/perfect weather .jpeg';
+    } else if (currentTemp >= 50) {
+        tempNumber.style.color ='black';
+        landscapeImg.src = 'assets/cool .jpeg';
+    } else {
+        tempNumber.style.color = 'teal';
+        landscapeImg.src = 'assets/Very Cold.jpg';
+    }
+}
+
+//Wave 3 & 4: 
+const currentCity = document.getElementById('current-city');
+const cityName = document.getElementById('city-name');
+
+cityName.addEventListener('input',()=>{
+    currentCity.textContent = cityName.value;
+    getCityRealTimeTemp(cityName.value);
+})
```

Comment:
Like stated in my other comment, these lines could be moved inside of the `DOMContentLoaded` event.
```suggestion
const currentCity = document.getElementById('current-city');
const cityName = document.getElementById('city-name');

cityName.addEventListener('input',() => {
    currentCity.textContent = cityName.value;
    getCityRealTimeTemp(cityName.value);
})
```